common/apache-poi/pom.xml

```diff
@@ -8,29 +8,44 @@
 	</parent>
 
 	<artifactId>org.apache.poi</artifactId>
-	<version>3.11.0-SNAPSHOT</version>
+	<version>3.17.0-SNAPSHOT</version>
 	<packaging>eclipse-plugin</packaging>
 
 	<dependencies>
 		<dependency>
 			<groupId>org.apache.poi</groupId>
 			<artifactId>poi</artifactId>
-			<version>3.11</version>
+			<version>3.17</version> <!-- latest 4.0.0 -->
```

Comment:
comments about latest are very relative. today it is latest, tomorrow it is not true anymore